src/components/input/_base.scss

```diff
@@ -28,9 +28,18 @@ $transitions-label-height: 1.2rem;
     cursor: pointer;
     z-index: $z-2;
 
+    &::placeholder {
+        color: transparent;
+        transition: color 250ms;
```

Comment:
Anyway i wonder if opacity works here so we don't have to define colours in the base scss